src/Platforms/AbstractPlatform.php

```diff
@@ -3882,6 +3896,28 @@ public function getTimeTypeDeclarationSQL(array $column)
         throw Exception::notSupported(__METHOD__);
     }
 
+    /**
+     * Adds a parameter to a TIME-type SQL declaration to indicate precision
+     *
+     * @param string  $decl   The SQL declaration to update e.g. TIME, DATETIME
+     * @param mixed[] $column The column definition
+     */
+    protected function addVariablePrecisionTimeSQL(string $decl, array $column): string
+    {
+        if (
+            ! isset($column['length']) ||
```

Comment:
Why is column precision defined by the "length" attribute?

Reply:
The PR description explains it:

> 'Precision' or 'scale' could have been used but existing documentation says these are reserved for DECIMAL types so 'length' was chosen as it seems to be parsed for all column types.

But "precision" would indeed look like a better fit imho. The documentation can (and should) be changed.

Reply:
Not really. Why do precision and scale default to anything for datetime fields at the moment? I think, we can change those defaults without a BC break as long as they remain the same for decaimals.

Reply:
I'll have another look as it's been a while.